HubSpot.NET/Api/EmailEvents/EmailCampaignListRequestOptions.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace HubSpot.NET.Api.EmailEvents
+{
+    /// <summary>
+    /// Options used when querying for email campaigns.
+    /// </summary>
+    public class EmailCampaignListRequestOptions
+    {
+        private int _limit = 20;
+
+        /// <summary>
+        /// Gets or sets the number of campaigns to return.
+        /// </summary>
+        /// <remarks>
+        /// Defaults to 20 which is also the hubspot api default. Max value is 100
+        /// </remarks>
+        /// <value>
+        /// The number of contacts to return.
+        /// </value>
+        public int Limit
+        {
+            get => _limit;
+            set
+            {
+                if (value < 1)
+                {
+                    throw new ArgumentException(
+                        $"Number of campaigns to return must be a positive ingeteger greater than 0 - you provided {value}");
```

Comment:
Minor typo on integer
```suggestion
                        $"Number of campaigns to return must be a positive integer greater than 0 - you provided {value}");
```